docs/reference/contributing/guidelines/workflow.md

```diff
@@ -93,13 +93,13 @@ We consider the following pull request types.
 
 #### Fix
 
-A bug fix is a backward-compatible internal change that fixes incorrect behavior.
+A bug fix is a change that fixes a specific defect in the codebase with backwards compatibility. These are the highest priority because of the positive impact the change will have on users developing against the same code. A bug fix should be limited to restoring the documented or proven otherwise, originally intended behavior. Bug fixes are candidates for patch releases.
```

Comment:
```suggestion
A bug fix is a change that fixes a specific defect in the codebase with backwards compatibility. These are the highest priority because of the positive impact the change will have on users developing against the same code. A bug fix should be limited to restoring the documented or proven otherwise, originally intended behavior. Bug fixes are candidates for patch releases. Large, non-trivial bug fixes approaching the size of refactors run the risk of being considered refactors themselves.

@0xc0170 

```

Reply:
+1, added